Makefile

```diff
@@ -1415,6 +1425,33 @@ musica_fortran_test:
 	$(eval MUSICA_FORTRAN_VERSION := $(shell pkg-config --modversion musica-fortran))
 	$(if $(findstring 1,$(MUSICA_FORTRAN_TEST)), $(info Built a simple test program with MUSICA-Fortran version $(MUSICA_FORTRAN_VERSION)), )
 
+scotch_fortran_test:
+	@#
+	@# Create a Fortran test program that will link against the SCOTCH library
+	@#
+	$(info Checking for a working MUSICA-Fortran library...)
```

Comment:
I think "MUSICA-Fortran" should be "Scotch" (or "PT-Scotch").

Reply:
Fixed